sdk/lib/core/map.dart

```diff
@@ -226,12 +226,10 @@ abstract class Map<K, V> {
   /**
    * Returns the value for the given [key] or null if [key] is not in the map.
    *
-   * Some maps allows keys to have `null` as a value,
-   * For those maps, a lookup using this operator does cannot be used to
-   * distinguish between a key not being in the map, and the key having a null
-   * value.
-   * Methods like [containsKey] or [putIfAbsent] can be use if the distinction
-   * is important.
+   * Some maps allows keys to have `null` as a value. For those maps, a lookup
```

Comment:
Also: "Some maps allow"
I'd prefer to retain the newline between "value" and "For". The separation into individual sentences, and sentences into stand-alone phrases when possible, per line, makes it easier to edit the documentation in the future without having to reflow (and when converted to HTML, extra whitespace doesn't matter anyway).